sys/decl.go

```diff
@@ -391,6 +391,8 @@ type IntType struct {
 	TypeCommon
 	TypeSize uintptr
 	Kind     IntKind
+	IsRange  bool
+	IntRange []uintptr
```

Comment:
randInt and rand64 return uintptr, but representation of that uintptr can mean negative numbers if treated as a signed integer (e.g. 0xffffffffffff means -1).

uintptr and int64 are pretty much the same as in C. uintptr is an unsigned integer with size equal to size of a pointer. int64 is a signed 64-bit integer.

OK, I think the simplest thing for now is to parse the numbers as signed, but then convert them to uintptr:
```
    var lo, hi int64
    if _, err := fmt.Sscanf(s, "%d-%d", &lo, &hi); err != nil {
        failf("failed to parse int range: %v (%v)", s, err)
    }
    fmt.Fprintf(out, "IntType{%v, Kind: IntRange, TypeSize: %v, rangeBegin: %v, rangeEnd: %v}", common(), typeToSize(typ), uintptr(lo), uintptr(hi))
```